### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by RuslanSky23**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+.product {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  max-width: 199px;
+  max-height: 407px;
+}
```
Comment (nazarmatsevych) [inline src/style.css:23]: It is not necessary to set max-height, remove this line

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  --main-text-color: #060b35;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  max-width: 199px;
```
Comment (nazarmatsevych) [inline src/style.css:21]: Change max-width to 200px and add box-sizing: border-box; rule, as the task description mentions the card width should be 200px including border

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
+/* stylelint-disable-next-line no-empty-source */
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (nazarmatsevych) [inline src/style.css:5]: image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  max-width: 199px;
+  max-height: 407px;
+}
+
+.title {
+  display: flex;
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 4px;
```
Comment (nazarmatsevych) [inline src/style.css:30]:
```suggestion
  margin: 40px 16px 4px;
```

replace with margin shorthand

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+  font-size: 12px;
+  line-height: 18px;
+  padding-left: 0.5px;
+}
+
+.image {
+  display: flex;
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
```
Comment (nazarmatsevych) [inline src/style.css:46]: Add padding to whole card instead of margins to each element

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+.stars__start {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 12px;
+  height: 12px;
+  padding: 0 7.5px 0 0;
+  margin-left: 0.5px;
```
Comment (nazarmatsevych) [inline src/style.css:85]: I think you can add background-position: center rule and remove this margin and backgrouind-size rules

Comment (nazarmatsevych) [inline src/style.css:85]: I think you can add background-position: center rule and remove this margin and backgrouind-size rules

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+.buybutton {
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  margin: 0 auto;
+  color: white;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  border-style: solid;
+  border-color: #00acdc;
```
Comment (nazarmatsevych) [inline src/style.css:118]: Move repeatable colors to variables

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+  width: 166px;
```
Comment (nazarmatsevych) [inline src/style.css:119]: Add padding to whole card and set width to 100%

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work,
pls check comments and proofread your code with [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work,
pls check comments and proofread your code with [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
test

**Comment by RuslanSky23** [comment]
https://RuslanSky23.github.io/layout_product-cards/

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  line-height: 16px;
+
+}
+
+.card__link:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
```
Comment (vlad-demchuk) [inline src/style.css:95]: move repeatable colors to variables

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,56 @@
...
+          $2,199
+        </div>
+      </div>
+      <div class="card__button">
+        <a
+        href="#"
+        class="card__link"
+        data-qa="hover">
+          Buy
+        </a>
```
Comment (anstsot) [inline src/index.html:51]: fix the code style
```suggestion
        <a
          href="#"
          class="card__link"
          data-qa="hover"
        >
          Buy
        </a>
```

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,56 @@
...
+    <link rel="stylesheet" href="./style.css">
+
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419"
+          class="card__image-img">
```
Comment (anstsot) [inline src/index.html:17]:
```suggestion
          class="card__image-img"
        >
```

**Review by anstsot — APPROVED** [review]
Good job, approved!

**Pull request closed without merging**